src/inspection/include/inspection/inspection.hpp

```diff
@@ -0,0 +1,67 @@
+#include <chrono>
+#include <functional>
+#include <memory>
+#include <string>
+#include <cmath>
+
+#include "custom_interfaces/msg/cone_array.hpp"
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/string.hpp"
+#include "fs_msgs/msg/control_command.hpp"
+#include "fs_msgs/msg/wheel_states.hpp"
+
+/**
+ * @class InspectionMission
+ * @brief Class responsible for the Inspection Mission of the car
+ *
+ * This class inherits from rclcpp::Node, subscribing to current velocity
+ * and ideal path topics, and publishing torque (or other output to the actuators).
```

Comment:
Ideal path topics?

Reply:
I detected that right after committing that, I copied that from other node and didn't notice that part; already corrected it.